app/src/app.js

```diff
@@ -39,7 +41,23 @@ function getInitialJukeboxState(accessToken) {
             [DEFAULT_PARAMS] : defaultParams 
           };
         }, bubbleUpError);
-    }, getEmptyInitialJukeboxState);
+    }, bubbleUpError);
+}
+
+/*
+ * Registers the user by adding them to DB.user.
+ * Returns a promise containing a 
+ */
+function registerUser(accessToken) {
+
+  // Connect to the DB
+  db.connect();
+
+  return getMe(accessToken).
+    then(function(userObject) {
+      id = userObject['body']['id'];
```

Comment:
yes - if fulfilled, userObject is a valid Spotify userObject. It therefore must have the speced out properties.